src/app.ts

```diff
@@ -32,7 +33,9 @@ main('gherkin');
 
 async function main(language: LanguageId) {
   const themeKey = 'vs'; //'vs' -> Light; 'vs-dark' -> Dark
-  const readOnly = true;
+  const readOnly = false;
```

Comment:
We have a problem here: we need to have the readonly monaco available somewhere for us to use within gx, then we'll have a different monaco to use with the web version (which I'll integrate with React for easier use) - I propose we make another branch off of `dev` and freeze it, we could also do the opposite and move this new editor with snippets and stuff to another branch and use it later for the integration with React